qutebrowser/browser/webkit/webkittab.py

```diff
@@ -781,6 +783,23 @@ def _on_frame_created(self, frame):
     def _on_contents_size_changed(self, size):
         self.contents_size_changed.emit(QSizeF(size))
 
+    @pyqtSlot()
+    def init_focustools(self):
+        """Initialize focustools.js in the main frame."""
+        self.run_js_async(
+            'window._qutebrowser = window._qutebrowser || {};')
+        self.run_js_async(
+            utils.read_file('javascript/focustools.js'))
+
+    @pyqtSlot()
+    def handle_clear_focus(self):
```

Comment:
I'm a bit confused here - why is this only here for QtWebKit?

Reply:
the analog for this function on the webengine side is `init_focustools` (and because webkit dosen't have a 'script' analog, we have to hook it into the page load. The naming is a bit inconsistent though, so I'll update it to match on both sides.